### DISCUSSION TITLE
Trucker caps

### DISCUSSION THREAD

**Discussion by ahoneybun** [elementary/store#150] [Released Products]
I think having an elementary hat would be very cool.

**Comment by ernmander** [comment]
> I think having an elementary hat would be very cool.

They already have done it. https://store.elementary.io/products/235661638

**Comment by ahoneybun** [comment]
That's not a trucker cap though, it's like this:

https://www.northerntool.com/shop/tools/product_200681586_200681586

**Reply by ernmander** [reply]
I did suggest the Trucker type cap, but they went with baseball https://github.com/elementary/store/discussions/118#discussioncomment-819108

**Comment by ekecso** [comment]
We now have a trucker cap: https://store.elementary.io/products/258646782
